### ISSUE TITLE
The question about additional adversarial loss ?

### ISSUE THREAD

**Issue by WhiteFu** [nipponjo/tts-arabic-pytorch#3]
Good Job! Looking forward to your repo!
 I also think that the (adv) version often sounds a bit clearer. but i have some question?
1.  Is it a joint training for tacotron2 and hifigan, or are there other details？ I'm curious about the fastpitch_ar_adv.pth model. 
2. When it is convenient to open source more code details, I would like to further verify the quality in other languages.

I'm ooking forward to your reply

**Comment by nipponjo** [comment]
Hello, thanks for your interest. I have pushed the files for training the FastPitch model with and without the adversarial loss to the scripts/ folder.

Some notes:
- More information about training the FastPitch model can be found at [NVIDIA/DeepLearningExamples](https://github.com/NVIDIA/DeepLearningExamples/tree/master/PyTorch/SpeechSynthesis/FastPitch)
- I used a DataLoader ('DynBatchDataset') that uses different batch sizes depending on the sequence lengths in the batch. The sizes are adapted to the graphics card I use (11 GB).
